x-pack/legacy/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer.js

```diff
@@ -1394,156 +1394,158 @@ export class TimeSeriesExplorer extends React.Component {
           jobs.length > 0 &&
           (fullRefresh === false || loading === false) &&
           hasResults === true && (
-            <EuiText className="results-container">
+            <div className="results-container-outer">
```

Comment:
IMHO we can remove the class name since there's no custom CSS tied to it. The comment below explains why it's necessary. Maybe it's worth adding something like `without padding` to `plain wrapping element`.

Reply:
Done in https://github.com/elastic/kibana/pull/54364/commits/ab9cee90fc0615543b7d9be2c3140b9bd9f88b14